piaf.opam

```diff
@@ -15,6 +15,7 @@ depends: [
   "ocaml-syntax-shims"
   "bigstringaf" {>= "0.5.0"}
   "httpaf"
+  "httpaf-lwt"
```

Comment:
do we need this? it's a dep of `httpaf-lwt-unix`

Reply:
nope, got rid of it.